### PR TITLE
add Tms, times() to posix module

### PR DESCRIPTION
**Pull request by disruptek**
Per https://pubs.opengroup.org/onlinepubs/9699919799/functions/times.html

### CODE REVIEW

**Comment by juancarlospaco** [comment]
Make the comments into documentation IMHO `#` :arrow_right: `##`. But looks good.

**Comment by disruptek** [comment]
@dom96 doesn't want posix documented manually, for... reasons.

**Comment by dom96** [comment]
> @dom96 doesn't want posix documented manually, for... reasons.

No, I don't want someone to spend their precious time copying and pasting docs from https://pubs.opengroup.org/onlinepubs/9699919799/.

File: lib/posix/posix.nim [hunk 1]
```diff
@@ -690,6 +690,12 @@ proc waitid*(a1: cint, a2: Id, a3: var SigInfo, a4: cint): cint {.
 proc waitpid*(a1: Pid, a2: var cint, a3: cint): Pid {.
   importc, header: "<sys/wait.h>".}
 
+type Tms* {.importc: "struct tms", header: "<sys/times.h>".} = object
+  tms_utime*, tms_stime*,            # User and system CPU time
+    tms_cutime*, tms_cstime*: Clock  # User and system CPU time of dead children
+
+proc times*(tms: ptr Tms): Clock {.importc, header: "<sys/times.h>", discardable.}
```
Comment (dom96) [inline lib/posix/posix.nim]: This is a wrapper, it shouldn't be discardable IMO.

Comment (disruptek) [reply lib/posix/posix.nim]: Alright, can I make `getrusage` un-discardable?

**Comment by disruptek** [comment]
I'm finding the name collision with `times` to be so annoying that I'm ready to just close this PR and settle for Rusage.  If you want it, or want to rename it, fine, but I pity the poor posix users otherwise.

File: lib/posix/posix.nim (hunk 1 above)
Comment (disruptek) [reply lib/posix/posix.nim]: And `wait`, I guess.

**Pull request closed without merging**